templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
Here's a previous Trello card on page objects, from 2013; we marked it as a Success: https://trello.com/c/CPilXWup/93-page-objects-in-test-suites

A lot of them look very similar to this.